Week-6/3Sumclosest.js

```diff
@@ -0,0 +1,18 @@
+var threeSumClosest = function (nums, target) {
```

Comment:
Please mention the time and space complexity as well in the comment.

Reply:
For all the programs

Reply:
added space and time complexity.
Pseudo code if i get time , i will be adding it..
Thanks for feedback